x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/indexlifecycle/CopyExecutionStateStep.java

```diff
@@ -22,7 +22,7 @@
  * new index has been created. Useful for actions such as shrink.
  */
 public class CopyExecutionStateStep extends ClusterStateActionStep {
-    public static final String NAME = "copy_execution_state";
+    public static final String NAME = "copy-execution-state";
```

Comment:
our other steps use kabob case, so this change is just to make it consistent with our other multi-word step names